packages/node_modules/@ciscospark/react-component-activity-menu/src/index.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import ActivityMenu from '.';
+
+describe(`ActivityMenu component`, () => {
+  const onExit = jest.fn();
+  const onMeet = jest.fn();
+  const onMessage = jest.fn();
+  const component = renderer.create(
+    <ActivityMenu
+      onExit={onExit}
+      onMeet={onMeet}
+      onMessage={onMessage}
+    />
+  );
+
+  it(`renders properly`, () => {
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
Ahhh right. I was thinking we can test side effects, but that doesn't quite work here. Given that this PR is pretty darn important. Maybe we start adding Enzyme based tests in a future feature and then dedicate some time going back through the components to do some cleanup and add tests.